packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -150,6 +177,7 @@
 				9740EEB11CF90186004384FC /* Flutter */,
 				97C146F01CF9000F007C117D /* Runner */,
 				680049182280D368006DD6AB /* image_picker_exampleTests */,
+				6801C8372555D726009DAF8D /* RunnerUITests */,
```

Comment:
This is a group, the thing that looks like a folder in the explorer on the left side.